frontend/src/bundles/studio/components/timeline/timeline.tsx

```diff
@@ -28,11 +30,35 @@ const Timeline: React.FC<Properties> = ({ playerRef }) => {
 
     const range = useAppSelector(({ studio }) => studio.range);
 
+    const onButtonClick = useCallback(
+        (type: RowType) => {
+            switch (type) {
+                case RowNames.SCENE: {
+                    dispatch(studioActions.addScene());
+                    dispatch(
+                        studioActions.setMenuActiveItem(MenuItems.AVATARS),
+                    );
+                    break;
+                }
+                case RowNames.SCRIPT: {
+                    dispatch(studioActions.addScript(NEW_SCRIPT_TEXT));
+                    dispatch(studioActions.setMenuActiveItem(MenuItems.SCRIPT));
+                    break;
+                }
+            }
+        },
+        [dispatch],
+    );
+
     const onResizeEnd = useCallback(
         (event: ResizeEndEvent) => {
             const activeItem = event.active.data.current;
             const activeItemType = activeItem['type'] as RowType;
 
+            if (activeItemType === RowNames.BUTTON) {
+                return;
+            }
+
             const updatedSpan = activeItem.getSpanFromResizeEvent?.(event);
 
             if (!updatedSpan || activeItemType === RowNames.SCRIPT) {
```

Comment:
```suggestion
            if (!updatedSpan || activeItemType === RowNames.SCRIPT || activeItemType === RowNames.BUTTON) {
```
We can put this condition here